crates/orchestrator/src/jobs/mod.rs

```diff
@@ -173,6 +173,21 @@ pub async fn verify_job(id: Uuid) -> Result<()> {
     Ok(())
 }
 
+/// Terminates the job and updates the status of the job in the DB.
+pub async fn handle_job_failure(id: Uuid) -> Result<()> {
```

Comment:
let's add a test case after the fixture PR is merged.

Reply:
Added.